ibm_catalog.json

```diff
@@ -484,11 +491,75 @@
                 "crn:v1:bluemix:public:iam::::serviceRole:Manager",
                 "crn:v1:bluemix:public:iam::::role:Editor"
               ],
-              "service_name": "event-notifications"
+              "service_name": "event-notifications",
+              "notes": "[Optional] Required if you are creating an Event Notifications Instance."
+            },
+            {
+              "service_name": "atracker",
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Writer",
+                "crn:v1:bluemix:public:iam::::role:Editor"
+              ],
+              "notes": "[Optional] Required if you are consuming Observability DA which creates Activity Tracker (AT) event routing instance."
```

Comment:
Activity Tracker event routing has nothing to do with Event Notifications. The integration point between Obseravility DA and EN DA right now is that user can pass a Cloud Monitoring CRN to force EN metrics to the Cloud Monitoring instance created by that DA. If user does not do this, metrics will be routed to whatever Cloud Monitoring instance is enabled with platform metrics, or whatever config the account metrics routing is set up with